src/core/pointcloud/qgspointcloudlayer.h

```diff
@@ -295,6 +295,8 @@ class CORE_EXPORT QgsPointCloudLayer : public QgsMapLayer, public QgsAbstractPro
     QgsPointCloudStatistics mStatistics;
     PointCloudStatisticsCalculationState mStatisticsCalculationState = PointCloudStatisticsCalculationState::NotStarted;
     long mStatsCalculationTask = 0;
+
+    bool mUsingDefaultRenderer = false;
```

Comment:
Shouldn't this get reset to false if an explicit call to setRenderer is made at any time?